ActiveLabel/ActiveLabel.swift

```diff
@@ -282,7 +279,7 @@ typealias ElementTuple = (range: NSRange, element: ActiveElement, type: ActiveTy
         addLinkAttribute(mutAttrString)
         textStorage.setAttributedString(mutAttrString)
         _customizing = true
-        text = mutAttrString.string
+        self.attributedText = mutAttrString
```

Comment:
`text` is an override over UILabel's property, so you need to set the text in there too in order for it to correctly forward the text.

Also, why is it that you stopped using it?